webui/react/src/pages/ExperimentDetails/ExperimentOverview.table.tsx

```diff
@@ -31,10 +32,11 @@ export const columns: ColumnType<TrialItem>[] = [
   },
   {
     key: 'bestValidation',
-    render: (_: string, record: TrialItem): ReactNode =>
-      record.bestValidationMetric && humanReadableFloatRenderer(record.bestValidationMetric),
```

Comment:
**question:** don't we want the tooltip when the metric value is out of "standard" range?

Reply:
we do I missed to copy this over from experimentDetails.tsx